openapi.json

```diff
@@ -319,6 +319,137 @@
         ]
       }
     },
+    "/clusters/{clusterId}/rules/{ruleId}/report": {
+      "get": {
+        "summary": "Returns the latest single report rule for the a cluster which contains information and content. Information about the org_id and user_id is taken from the token.",
```

Comment:
report rule -> rule report

Reply:
Returns the latest single rule report for a cluster that contains information and content. Information about the org_id and user_id is taken from the token.